clients/identity/authentication.go

```diff
@@ -27,24 +34,28 @@ func NewAPIClient(cli common.RCHTTPClienter, url string) (api *IdentityClient) {
 
 // CheckRequest calls the AuthAPI to check florenceToken or authToken
 func (api IdentityClient) CheckRequest(req *http.Request) (context.Context, int, error) {
-
+	var logData log.Data
 	log.DebugR(req, "CheckRequest called", nil)
 
 	ctx := req.Context()
 
 	florenceToken := req.Header.Get(common.FlorenceHeaderKey)
 	authToken := req.Header.Get(common.AuthHeaderKey)
+	logData = splitTokens(florenceToken, authToken)
 
 	isUserReq := len(florenceToken) > 0
 	isServiceReq := len(authToken) > 0
+	logData["is_user_request"] = isUserReq
+	logData["is_service_request"] = isServiceReq
```

Comment:
it feels like this logic could be tidied up, as the two request types are mutually exclusive. the `if both nil` check could be moved up to before line 44, we dont need to split the tokens if they're nil, and the check can be done based on length.

Reply:
I've moved the `logData` to when it's needed.